src/conn/conn_api.c

```diff
@@ -9,6 +9,7 @@
 #include "wt_internal.h"
 
 static int __conn_statistics_config(WT_SESSION_IMPL *, const char *[]);
+static int __conn_verbose_cmd(WT_SESSION_IMPL *, const char *);
```

Comment:
> I've got some concerns about how `WT_CONNECTION::reconfigure()` with `verbose` is being used and seems to be being expanded moving forward.

I prefer:
```
WT_CONNECTION.reconfigure("verbose=(dump_txn,dump_cache)"))
```
to the previous:
```
WT_CONNECTION.reconfigure("diagnostic=(dump_txn,dump_cache)")
```
having this functionality in "verbose" makes sense to me. That said, I don't have a strong opinion, I'm happy to go with whatever you and @michaelcahill decide.